awx/main/analytics/collectors.py

```diff
@@ -262,6 +264,7 @@ def _copy_table(table, query, path):
                               main_jobevent.event_data::json->'res'->'warnings' AS warnings,
                               main_jobevent.event_data::json->'res'->'deprecations' AS deprecations
                               FROM main_jobevent 
+                              LEFT JOIN main_inventory ON main_inventory.id = main_job.inventory_id
```

Comment:
This seems to be a typo, it looks like we're dealing with `main_jobevent` in this query

Reply:
ope, that's what I get for typing too fast, :+1: